smug.go

```diff
@@ -132,7 +132,12 @@ func (smug Smug) Start(config Config, windows []string) error {
 	}
 
 	if len(windows) == 0 {
-		err = smug.tmux.KillWindow(ses + "0")
+		windows, err := smug.tmux.ListWindows(ses)
+		if err != nil {
+			return err
+		}
+
+		err = smug.tmux.KillWindow(ses + windows[0])
```

Comment:
When I ran the string split here, I was getting errors because the output of [0] would return a string integer like `'1'` which is why I put the unquote function in my code snippet. Could you clarify why this word without `unquote` vs what I posted? Purely educational.

Reply:
I think in your snippet you got quoted output like `'1'` because you specified quotes in the parameters for `list-windows`:  `exec.Command("tmux", "list-windows", "-t", target, "-F'#{window_index}'")`

In this PR I've removed them: `exec.Command("tmux", "list-windows", "-t", target, "-F", "#{window_index}")`, so it returns plain numbers.

Reply:
Dang! I totally missed that, thank you!